frontend/styles/custom.css

```
@@ -48,4 +48,35 @@ header input {
 }
 .hgf-scrollbar::-webkit-scrollbar-track {
   background: white;
-}
\ No newline at end of file
+}
+
+#research-fields-container {
+  display: grid;
+  grid-template: 1fr / 1fr;
+  place-items: center;
+  z-index: 1;
+}
+
+#research-fields-container > * {
+  grid-column: 1 / 1;
+  grid-row: 1 / 1;
+}
+
+#research-fields-container #research-fields-list {
+  z-index: 3;
+}
+
+#research-fields-container .research-field-image {
+  z-index: 2;
+  opacity: 0;
+  transition: opacity 0.3s ease 0.1s;
```

```suggestion
  transition: opacity 0.3s ease 0s;
```

I remember that the delay in the transition was a fix to solve some kind of flickering or so. To me, the 0.1s feels to long still, and I tried it with 0s. For me it looks good. What do you think?

---

ok, we can remove the delay :+1: